public/review.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class Foo {
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)
* Opening brace of a class must be on the line after the definition

<!-- Comment by [SideCI](https://sideci.com) -->